stories/assembly-aap-gcp-backup-and-recovery.adoc

```diff
@@ -1,10 +1,18 @@
 ifdef::context[:parent-context: {context}]
 
 [id="assembly-aap-gcp-backup-and-recovery"]
-= Backup and Recovery
+= Backup and Restore
 
 :context: aap-gcp-backup-recovery
 
 include::topics/proc-aap-gcp-backup-process.adoc[leveloffset=+1]
```

Comment:
There's a Procedure Overview in the googledoc which ought to be in here, so you know what's coming.